### ISSUE TITLE
Re-add `aria-activedescendant="ucfhb-autocomplete-selected"` to search input dynamically

### ISSUE THREAD

**Issue by cjg89** [UCF/UCF-Header#16]
The search input had the `aria-activedescendant="ucfhb-autocomplete-selected"` attribute applied to it until recently, when I realized it was not validating due to `#ucfhb-autocomplete-selected` not being an element that is always present on the page.

This attribute should be applied to the search input (`#ucfhb-search-field`) only when an element with the ID `#ucfhb-autocomplete-selected` exists (when autosuggest results are present), and will need to be removed from the search input whenever the autosuggestion list is empty.

**Comment by cjg89** [comment]
In my testing, adding the `aria-activedescendant` attribute correctly results in a double readout (at least in ChromeVox) of the highlighted autosuggestion when it's selected via up/down keypress, even after removing the [logic that moves the cursor to force a readout of the search input value](https://github.com/UCF/UCF-Header/blob/master/compiler/assets/university-header-templ.js#L591-L618).  Because of this, I think it's best that we omit the attribute entirely to keep the spoken feedback clean+minimal.  Closing task.
